keps/sig-api-machinery/20200309-consistent-resource-versions-semantics.md

```diff
@@ -0,0 +1,216 @@
+---
```

Comment:
nit: I think we're moving it to a separate file as per recent KEP template adjustments.